src/liedle/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Liedle</title>
+    <link rel="stylesheet" href="../main.css">
+    <link rel="stylesheet" href="liedle.css">
+</head>
+<body>
+    <div class="content">
+        <div class="header">Liedle</div>
+        <div data-grid id="grid" class="grid"></div>
```

Comment:
I think the `data-grid` attribute on this line is redundant, since there's no value specified. 
```suggestion
        <div id="grid" class="grid"></div>
```

Reply:
Nice catch! I can't remember why I had that there 😅.